### PR TITLE
Update remarks hint text

### LINKED ISSUE #2489: Rewording the hints under "Remarks" in the competition forms
While a new form is implemented with specific checkboxes for the article Z, can we please have this text...

https://github.com/thewca/worldcubeassociation.org/blob/6f8ce63120916716b8d750841df8c8625f69b6b8/WcaOnRails/config/locales/en.yml#L366

...reworded to:
Some additional information you want to pass on to the board. For example, elaborate your reasons if this competition is requested less than four weeks away, or if you are requesting the application of Article Z (***LINK***). Please fill this out in English!

### PR DESCRIPTION
**Pull request by pedrosino**
Fixes #2489

### CODE REVIEW

File: WcaOnRails/config/locales/en.yml [hunk 1]
```diff
@@ -363,7 +363,7 @@ en:
         organizer_ids: "Competition organizers (they need an account to be listed here)."
         external_website: "The website of the competition. Must be a valid http(s) url."
         registration_open: "Note: Registration open and close are in UTC time"
-        remarks: "Some additional information you want to pass on to the board. For example, reasons why this competition was requested less than one month away, if applicable. Please fill this out in English!"
+        remarks: "Some additional information you want to pass on to the board. For example, elaborate your reasons if this competition is requested less than four weeks away, or if you are requesting the application of Article Z (https://www.worldcubeassociation.org/regulations/#article-Z-optional). Please fill this out in English!"
```
Comment (jfly) [inline WcaOnRails/config/locales/en.yml]: Could you include a screenshot of this? Is the user going to have to copy paste this link in order to view it? Could we make this an actual html link somehow?

**Comment by pedrosino** [comment]
I tried to make it an actual link, but could not. I didn't find the part where the key is called - there's just a f.input "remarks" so I wasn't sure how to make the link.

**Comment by jfly** [comment]
Yeah, I'm afraid this is buried deep inside of simpleform somewhere. Unfortunately, I think you have to explicitly pass the hint in when calling `f.input` (see https://github.com/plataformatec/simple_form/issues/965). I believe we actually already do this in a few other place in our codebase. You shouldn't have to call `raw` as they do in the example, just rename the key from `remarks` to `remarks_html` and it should automatically become a `SafeBuffer`.

**Comment by pedrosino** [comment]
Another attempt, but not still quite there:

image

I tried doing it the same way as https://github.com/thewca/worldcubeassociation.org/blob/master/WcaOnRails/app/views/competitions/_competition_form.html.erb#L34 but maybe got it wrong...

**Comment by pedrosino** [comment]
Weirdly enough, I just booted it up locally and it was working.
I had the link_to structure wrong, but fixed and now it looks as expected:
image

@Luis-J-Ianez is this what you envisioned?

**Comment by Luis-J-Ianez** [comment]
Yes, @pedrosino! Thanks.

**Comment by pedrosino** [comment]
argh, I messed up the branches and pushed the wrong stuff here :(

I have it at home, so will push again later.

**Comment by pedrosino** [comment]
Fixed and merging.

**Pull request merged**